tensornetwork/backends/jax/jitted_functions_test.py

```diff
@@ -24,7 +24,8 @@ def matvec(vector, matrix):
   kv = jax.numpy.zeros((ncv + 1, D), dtype=dtype)
   H = jax.numpy.zeros((ncv + 1, ncv), dtype=dtype)
   start = 0
-  kv, H, it, _ = arnoldi(matvec, [mat], x, kv, H, start, ncv, 0.01)
+  precision = jax.lax.Precision.HIGHEST
+  kv, H, it, _ = arnoldi(matvec, [mat], x, kv, H, start, ncv, 0.01, precision)
```

Comment:
The hardcoded 0.01 is a bit weird

Reply:
Or did I write that?

Reply:
That was me, its a threshold value. Added a variable name